image-segmentation/workflow_DGXA100_40GB_4.yaml

```diff
@@ -0,0 +1,204 @@
+
+          cd $(workspaces.pvc.path)/KiTS19
+          rm -rf raw-data-dir
+
+---
+
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: WorkflowTemplate
+metadata:
+  name: mlperf-segmentation-train
+spec:
+  params:
+    - default: mlperf
+      description: Name of the PVC that stores dataset
+      name: pvc_name
+  resource:
+    failureRules:
+      fieldSelector: status.phase==Failed
+    manifest: |
+      apiVersion: batch.tensorstack.dev/v1beta1
+      kind: MPIJob
+      metadata:
+        name: mlperf-segmentation
+      spec:
+        mca:
+          btl: ^openib
+          pml: ob1
+        mpiHome: /usr/local/mpi
+        ssh:
+          sshdPath: /usr/sbin/sshd
+        scheduler:
+          t9kScheduler:
+            queue: mlperf
+            priority: 50
+        runPolicy:
+          cleanUpWorkers: true
+        worker:
+          replicas: 4
+          processesPerWorker: 1
+          processRecovery:
+            enable: true
+            limit: 0
+          cmd:
+            - python
+            - main.py
+            - "--data_dir=/pvc/KiTS19/data"
+            - "--epochs=10000"
+            - "--quality_threshold=0.908"
+            - "--batch_size=4"
+            - "--evaluate_every=20"
+            - "--start_eval_at=1000"
+            - "--lr_warmup_epochs=1000"
+            - "--optimizer=nag"
+            - "--learning_rate=2.0"
+            - "--static_cast"
+            - "-sls=32784"
+            - "-gpf=4"
+            - "--warmup"
+            - "--val_batch_size=1"
+            - "--num_workers=4"
+            - "--input_batch_multiplier=4"
+            - "--gpu_per_node=1"
+            - "-ucl"
+            - "-sts"
+            - "--fp16in"
+          template:
+            spec:
+              containers:
+                - name: mpi-worker
+                  image: registry.tensorstack.cn/t9k/mlperf-nvidia:image_segmentation
+                  workingDir: /workspace/unet3d/
+                  env:
+                    - name: LD_LIBRARY_PATH
+                      value: "/usr/local/cuda/compat/lib:/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/usr/local/lib"
+                    - name: OMP_NUM_THREADS
+                      value: "1"
+                    - name: HOROVOD_CYCLE_TIME
+                      value: "0.1"
+                  resources:
+                    limits:
+                      cpu: 4
+                      memory: 8Gi
+                      tensorstack.dev/nvidia-gpu: 1
+                  volumeMounts:
+                    - mountPath: /pvc
+                      name: data
+                    - mountPath: /dev/shm
+                      name: dshm
+                    - mountPath: /results
+                      name: results
+              volumes:
+                - name: data
+                  persistentVolumeClaim:
+                    claimName: mlperf
```

Comment:
PVC name `mlperf` is hard-coded in this MPIJob.  This poses a problem as  changing the value of the parameter `pvc_name` in workflowrun `mlperf-segmentation` won't work.

Reply:
fixed to `$(params.pvc_name)`